src/client/components/map/Map.tsx

```diff
@@ -230,6 +248,35 @@ const DistrictsMap = ({
       );
   };
 
+  // Compute array of top geolevels split across multiple districts
+  const splitCountiesDistricts = project?.districtsDefinition.map(c => {
+    if (Array.isArray(c)) {
+      return c;
+    } else {
+      return undefined;
+    }
+  });
+
+  // Set features from topmost geolayer that are split across districts to be selected
+  // eslint-disable-next-line
+  const counties =
+    map &&
```

Comment:
We never use counties?
```suggestion
  map &&
```